test/test_proxyweb.py

```
@@ -2210,6 +2219,313 @@ def run(self, suite):
+        "SELECT variable_name FROM global_variables ORDER BY variable_value DESC LIMIT {n}",
+        "SELECT variable_value FROM global_variables ORDER BY variable_value DESC LIMIT {n}",
+        "SELECT variable_name, variable_value FROM global_variables ORDER BY variable_value DESC LIMIT {n}",
+        "SELECT COUNT(*) FROM global_variables WHERE variable_name LIKE 'mysql%%' LIMIT {n}",
+        "SELECT COUNT(*) FROM global_variables WHERE variable_name LIKE 'admin%%' LIMIT {n}",
+        "SELECT COUNT(*) FROM global_variables LIMIT {n}",
+        "SELECT variable_name FROM global_variables WHERE variable_value != '' LIMIT {n}",
+        "SELECT variable_value FROM global_variables WHERE variable_value != '' LIMIT {n}",
+        "SELECT variable_name FROM global_variables WHERE variable_name LIKE 'admin%%' ORDER BY variable_name LIMIT {n}",
+        "SELECT variable_value FROM global_variables WHERE variable_name LIKE 'admin%%' ORDER BY variable_name LIMIT {n}",
+        "SELECT variable_name, variable_value FROM global_variables WHERE variable_value != '' ORDER BY variable_name LIMIT {n}",
+    ]
```

Ruff (0.15.6)

[warning] Mutable default value for class attribute

(RUF012)